src/components/AppNavigationForm.vue

```diff
@@ -110,6 +119,19 @@ export default {
 			return 'icon-forms'
 		},
 
+		/**
+		 * Check if form is current form and set active
+		 */
+		isActive() {
+			if (this.form.hash === this.$route.params.hash) {
+				return true
+			}
+			return false
```

Comment:
A bit shorter & cleaner, if you directly return the result:
```suggestion
			return this.form.hash === this.$route.params.hash
```